demo/billing/resources/views/billing_batches.sql.in

```diff
@@ -1,7 +1,7 @@
 CREATE MATERIALIZED VIEW billing_batches AS
 SELECT
     billing_raw_data.id,
-    to_timestamp((billing_raw_data.interval_start->'seconds')::int) interval_start,
-    to_timestamp((billing_raw_data.interval_end->'seconds')::int) interval_end
+    to_timestamp((billing_raw_data.interval_start->'seconds')::bigint) interval_start,
+    to_timestamp((billing_raw_data.interval_end->'seconds')::bigint) interval_end
```

Comment:
I thought we supported the timestamp well-known type now? We'd need to change the billing demo to emit it, though.

Reply:
I think that's what we are using but I think everything gets converted to strings because its all a bunch of nested objects that get parsed as jsonb